### ISSUE TITLE
Syntax Highlighting

### ISSUE THREAD

**Issue by daniel-wing** [victoriadrake/hugo-theme-introduction#127]
I am using the theme through RStudio Blogdown package and I can't get the Syntax Highlighting working. I have tried every Param config I can think of and even tried to add a js.html in the patials/js folder to no success.

Is it just me? Any ideas on how to get it working?

**Comment by hanzei** [comment]
Could you share the code of your website?

**Comment by victoriadotdev** [comment]
Hey @daniel-wing,

If this remains an issue please ensure `pygmentsCodeFences = true` and other syntax highlighting variables are set in your config file at the first level, and not under `[params]` or equivalent. Example `config.toml`:

```
baseURL = "http://example.com/"
theme = "introduction"
DefaultContentLanguage = "en"
pygmentsCodeFences = true

[params]
    themeStyle                   = "light"
...
```

Closing as stale for now.

**Comment by Athanasiamo** [comment]
Hi. 

I'm having issues with this at the moment. Been using this theme for my page a while, but trying to adapt to the theme updates, which broke some stuff.


```
baseURL                          = "http://example.com/"  # Your domain name. Must end with "/"
theme = "hugo-theme-introduction"
DefaultContentLanguage           = "en"                    # Default language for multilingual sites
pygmentsStyle       = "autumn"                    
pygmentsCodeFences  = true 
highlightStyle      = "monokai"     

[params]
    themeStyle                   = "light"                 # Choose "light" or "dark"
```

but I'm not getting code highlights :/

**Comment by victoriadrake** [comment]
Hi! Are you using [code fencing](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting) properly? It should look like this in your markdown files:

```markdown
```JavaScript
  const thisIsATest = () => {
    console.log("This is funny")
  }
```

Otherwise, I can be of more help if you link to your repo.

**Comment by Athanasiamo** [comment]
Hi. 

Sorry for leaving this, I did not have time to actually fix the issue when I was trying last. Finally had some time to work with it, so this is what I have found so far.

Both I and the original poster are using the R-package blogdown for the website. This is where the issue is arising I believe. Since blogdown renders html, the code-fencing is actually not there for hugo to work with. This means that setting `pygmentsCodeFences  = true ` will not have any effect on pages rendered through blogdown. 

This was _not_ the case before integrated syntax-highlighting in hugo, so it really is a shame. 

I am guessing this arises from what is discussed in a [previous PR](https://github.com/victoriadrake/hugo-theme-introduction/pull/26) with Bulma and Chroma colliding.
